src/components/PopupWithForm.js

```diff
@@ -25,16 +25,16 @@ class PopupWithForm extends Popup {
     this._handleFormSubmit(inputValues);
   };
 
-  showLoading(isLoading) {
+  renderLoading(isLoading) {
     if (isLoading) {
-      this._submitButton.textContent = "Saving...";
+      this._popupForm.querySelector(".modal__button").textContent = "Saving...";
```

Comment:
Use `this._submitButton` instead of ` this._popupForm.querySelector(".modal__button")`, since it contains the same element